aries-site/src/pages/index.js

```diff
@@ -22,34 +21,18 @@ const HomeTiles = ({ ...rest }) => {
   );
 };
 
-// Needs to be <a> in DOM for web crawling: https://support.google.com/webmasters/answer/9112205?hl=en
-const AnchorTile = forwardRef(({ ...rest }, ref) => (
-  <Tile as="a" fill ref={ref} style={{ textDecoration: 'none' }} {...rest} />
-));
-
-// Reasoning for using forwardRef: https://nextjs.org/docs/api-reference/next/link#example-with-reactforwardref
-const TopicTile = forwardRef(({ topic, ...rest }, ref) => {
-  return (
-    <AnchorTile
-      pad="medium"
-      background={topic.color}
-      key={topic.color}
-      ref={ref}
-      {...rest}
-    >
-      <TileContent
-        key={topic.name}
-        title={topic.name}
-        subTitle={topic.description}
-        icon={topic.icon('xlarge')}
-      />
-    </AnchorTile>
-  );
-});
-
 const title = 'Home';
 const pageDetails = getPageDetails(title);
-const topicList = pageDetails.pages.map(topic => getPageDetails(topic));
+
+const cards = structure
+  .map(obj => {
+    const page = obj;
+    const parent = getParentPage(page.name);
+    page.parent = parent;
+    return page;
+  })
+  .filter(page => page.parent !== undefined)
+  .filter(page => page.parent.name !== 'Home');
```

Comment:
Currently, some topics such as "Principles" that are now under Foundation were previously under Guidelines (which seems to be a deprecated topic) -- we'll need to move the Guidelines `pages` into the Foundation `pages` in the structure file as well as the file folder structure.

Reply:
Yes. There is going to be a decent amount of re-org on `structure.js`, but was trying to keep it somewhat isolated for now. Some tests are specifically targeting `Guidelines`, so I kept some of that in place for now. Was hoping you or @britt6612 could take a look at tests since you are both more familiar.

Reply:
Gotcha -- if we want to update these pages to now be under "Foundation", I can push a PR to this branch that updates those tests. Let me know if you'd like that.

Reply:
That'd be great.